gulp/tasks/test.js

```diff
@@ -111,6 +122,17 @@ gulp.task( 'test:coveralls', function() {
     .pipe( $.coveralls() );
 } );
 
+// This task will only run on Travis
+gulp.task( 'test:a11y', function() {
+  spawn(
+    fsHelper.getBinary( 'wcag', '.bin' ),
+    _getWCAGParams(),
+    { stdio: 'inherit' } )
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/2ec4d5b08584b32f28420a5429edda0eb067a77e